src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +81,17 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ResponseEntity processJSONException(final JSONException e, final NativeWebRequest nativeWebRequest) {
+        if (e.getCause() == null) {
```

Comment:
Because in some case it will have information which is not relevant to the user of the request and won't be informative for the user at all. I think here we want to show the user the error which is related to the json formatting and not internal errors inside json library.